include/boost/beast/http/impl/fields.hpp

```diff
@@ -990,6 +1068,9 @@ void
 basic_fields<Allocator>::
 set_chunked_impl(bool value)
 {
+    using A = typename beast::detail::allocator_traits<
+        Allocator>::template rebind_alloc<char>;
+    detail::temporary_buffer<A> buf{A{this->get()}};
```

Comment:
We should not use the allocator of the `basic_fields` for the temporary string. Instead, we should simply use the system allocator (`std::allocator<char>`).